source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/AdaptiveCardRenderer.java

```diff
@@ -242,6 +244,9 @@ else if (alignment == ActionAlignment.Center.swigValue())
             actionButtonsLayout.setOrientation(LinearLayout.HORIZONTAL);
         }
 
+        Spacing spacing = hostConfig.getActions().getSpacing();
+        BaseCardElementRenderer.setSpacingAndSeparator(context, viewGroup, spacing, false, hostConfig, true);
```

Comment:
Can you add a comment next to the false and true values to know what do they stand for?

Reply:
Yes. Will add the same comment as has been used  elsewhere .